packages/web-pkg/src/composables/actions/files/useFileActions.ts

```diff
@@ -276,6 +276,11 @@ export const useFileActions = ({ store }: { store?: Store<any> } = {}) => {
       return []
     }
 
+    // enforce open url files with system actions
+    if (options.resources[0].extension === 'url') {
```

Comment:
.url files will have 'application/ocet-stream' as mimetype. Which is assigned to Collabora and OnlyOffice, therefore url files will be opened with respective editors. 

We might fix it in the frontend with this code or need to ask OCIS to return another mime-type like  
https://github.com/owncloud/web/issues/9968